utils/build/docker/golang/parametric/datadog.go

```diff
@@ -101,6 +107,62 @@ func (s *apmClientServer) SpanSetError(ctx context.Context, args *SpanSetErrorAr
 	return &SpanSetErrorReturn{}, nil
 }
 
+type CustomLogger struct {
+	*logrus.Logger
+}
+
+func (l *CustomLogger) Log(logMessage string) {
+	re := regexp.MustCompile(`DATADOG TRACER CONFIGURATION (\{.*\})`)
+	matches := re.FindStringSubmatch(logMessage)
+	if len(matches) < 2 {
+		log.Printf("JSON not found in log message")
+		return
+	}
+	jsonStr := matches[1]
+
+	var config map[string]interface{}
+	if err := json.Unmarshal([]byte(jsonStr), &config); err != nil {
```

Comment:
Nit: instead of marshaling this json to a map we can use a struct: https://medium.com/@prashant2018/parse-json-string-to-struct-object-in-golang-59d3cf6bce7. This way we can be very intentional about the fields we capture